### ISSUE TITLE
Define site colours using css variables

### ISSUE THREAD

**Issue by hyanwong** [tskit-dev/tskit-site#15]
It would be nice to automatically be able to colour SVG diagrams using the site colours. This is a bit tricky because the SVG files contain their own stylesheets, which are not accessible to sass processing. 

However, we *can* use css variables to do this. So in main stylesheet we would say something like:

```
:root {
    --main-colour:#06c;
}
```

(where we substitute `#06c` with whatever sass colour we want).

Then in the SVG (e.g. of a tree sequence) I can simply do:

```
<style>.axis line {stroke: var(--main-colour)}</style>
```

I can't see any down sides to this, as it doesn't affect anything else. The only issue is if it isn't supported in modern browsers (see https://caniuse.com/css-variables). And what names we should use for the site colours.

**Comment by benjeffery** [comment]
Wow, didn't realise CSS variables had come along so much in compat. IE11 is the only hold out. As long as IE11 is still readable even if it is ugly then I say do it.

**Comment by benjeffery** [comment]
Closed by #18
